section2/exercises/if-statements.js

```diff
@@ -44,7 +44,14 @@ Log to the console the appropriate age range
 var dogAge = 3;
 // Write your conditional here
 
-
+if (dogAge <= 2) {
```

Comment:
Good solution for this challenge!

As far as the details, what we refer to as "style and syntax" - make sure you are following Ruby conventions - 
- add a space between `>=` and `10` on line 49
- the `else {` should be up on the same line as the previous closing curly bracket
- the `console.log("Adult") is indented one too many times

SUPER nit picky things, but they add up in a large codebase, and you want to make your code as easily readable as possible.